plugins/modules/helm.py

```diff
@@ -0,0 +1,494 @@
+    with ` --username` and `--password` before calling the module
+
+options:
+  binary_path:
+    description:
+      - The path of a helm binary to use.
+    required: false
+    type: path
+  chart_ref:
+    description:
+      - chart_reference on chart repository.
+      - path to a packaged chart.
+      - path to an unpacked chart directory.
+      - absolute URL.
+      - Required when I(release_state) is set to C(present).
+    required: false
+    type: path
+  chart_repo_url:
+    description:
+      - Chart repository URL where to locate the requested chart.
+    required: false
+    type: str
+  chart_version:
+    description:
+      - Chart version to install. If this is not specified, the latest version is installed.
+    required: false
+    type: str
+  release_name:
+    description:
+      - Release name to manage.
+    required: true
+    type: str
+    aliases: [ name ]
+  release_namespace:
+    description:
+      - Kubernetes namespace where the chart should be installed.
+    required: true
+    type: str
+    aliases: [ namespace ]
+  release_state:
+    choices: ['present', 'absent']
+    description:
+      - Desirated state of release.
+    required: false
+    default: present
+    aliases: [ state ]
+    type: str
+  release_values:
+    description:
+        - Value to pass to chart.
+    required: false
+    default: {}
+    aliases: [ values ]
+    type: dict
+  update_repo_cache:
+    description:
+      - Run C(helm repo update) before the operation. Can be run as part of the package installation or as a separate step.
+    default: false
+    type: bool
+
+#Helm options
+  disable_hook:
+    description:
+      - Helm option to disable hook on install/upgrade/delete.
+    default: False
+    type: bool
+  force:
+    description:
+      - Helm option to force reinstall, ignore on new install.
+    default: False
+    type: bool
+  kube_context:
+    description:
+      - Helm option to specify which kubeconfig context to use.
+    type: str
+  kubeconfig_path:
+    description:
+      - Helm option to specify kubeconfig path to use.
+    type: path
+    aliases: [ kubeconfig ]
+  purge:
+    description:
+      - Remove the release from the store and make its name free for later use.
+    default: True
+    type: bool
+  wait:
+    description:
+      - Wait until all Pods, PVCs, Services, and minimum number of Pods of a Deployment are in a ready state before marking the release as successful.
+    default: False
+    type: bool
+  wait_timeout:
+    description:
+      - Timeout when wait option is enabled (helm2 is a number of seconds, helm3 is a duration).
+    type: str
+'''
+
+EXAMPLES = '''
+- name: Create helm namespace HELM 3 doesn't create it automatically
+  k8s:
+    api_version: v1
+    kind: Namespace
+    name: "monitoring"
+    wait: true
+
+# From repository
+- name: Add stable chart repo
+  shell: "helm repo add stable https://kubernetes-charts.storage.googleapis.com"
```

Comment:
Just wondering—should we add a module that allows managing chart repos more easily? Using shell is not idempotent unless the user does a lot of extra work. This could be a follow-up at some point.

Reply:
(Ref @tatemz's comment from earlier in this PR thread)

> I'm testing this out locally and notice that we might need a way to add helm repos before updating the cache.

Reply:
I added a module to manage helm repository and update examples and tests